src/reducers/change-fixture.js

```diff
@@ -0,0 +1,10 @@
+var CHANGE_FIXTURE = require('../actions/change-fixture.js').type;
+
+module.exports = function(state = {}, action) {
+  switch (action.type) {
+    case CHANGE_FIXTURE:
+      return action.data || {};
```

Comment:
In that case, the reducer behaves correctly, as the redux store should be a single object. Sending `true` or any non-object truthy value might be a problem here, so I guess we can check if `action.data` is an object, and return `{}` otherwise.